workflows/sample/tasks/hifiasm_trio.wdl

```diff
@@ -45,11 +45,10 @@ task hifiasm_trio_assemble {
     File hap2_p_ctg        = "~{prefix}.dip.hap2.p_ctg.gfa"
     File hap2_p_ctg_lowQ   = "~{prefix}.dip.hap2.p_ctg.lowQ.bed"
     File hap2_p_noseq      = "~{prefix}.dip.hap2.p_ctg.noseq.gfa"
-    File p_utg             = "~{prefix}.dip.p_utg.gfa"
-    File r_utg             = "~{prefix}.dip.r_utg.gfa"
```

Comment:
These are the raw and polished unitigs, before the assembly is phased into haplotypes.  These files also include all of the FASTA sequence as well.  I prefer to keep the `noseq` versions of these files, so you still have the graph if you want to investigate how the unitigs were split into the phased haplotypes.